internal/connector/pingone/sso/resources/pingone_application_role_assignment.go

```diff
@@ -90,10 +90,11 @@ func (r *PingoneApplicationRoleAssignmentResource) ExportAll() (*[]connector.Imp
 
 			for _, roleAssignment := range appRoleAssignmentsEmbedded.GetRoleAssignments() {
 				roleAssignmentId, roleAssignmentIdOk := roleAssignment.GetIdOk()
-				if roleAssignmentIdOk {
+				roleAssignmentRoleName, roleAssignmentRoleOk := roleAssignment.GetRoleOk()
```

Comment:
If we do this here, the above logic to check the AccessControl role is not needed

Reply:
name